internal/services/cosmos/cosmosdb_account_resource.go

```diff
@@ -808,7 +884,7 @@ func resourceCosmosDbAccountUpdate(d *pluginsdk.ResourceData, meta interface{})
 			IsVirtualNetworkFilterEnabled:      utils.Bool(isVirtualNetworkFilterEnabled),
 			EnableFreeTier:                     utils.Bool(enableFreeTier),
 			EnableAutomaticFailover:            utils.Bool(enableAutomaticFailover),
-			Capabilities:                       expandAzureRmCosmosDBAccountCapabilities(d),
+			Capabilities:                       &oldCapabilities,
```

Comment:
As it stands this will overwrite the just updated value, however we can retrieve the CosmosDB Account from the API once it's updated, and then reuse this value from the `read` property:

```suggestion
			Capabilities:                       resp.Properties.Capabilities,
```

Reply:
Fixed.